<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Renderable_Dlg</class>
 <widget class="QWidget" name="Renderable_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Subsets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget_subsets">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_updateFromDrawable">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="D3DChips_Dlg.qrc">
            <normaloff>:/D3DChips_Dlg/Resources/arrow_circle_double.png</normaloff>:/D3DChips_Dlg/Resources/arrow_circle_double.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ssUp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="D3DChips_Dlg.qrc">
            <normaloff>:/D3DChips_Dlg/Resources/Stock-Index-Up-icon16.png</normaloff>:/D3DChips_Dlg/Resources/Stock-Index-Up-icon16.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ssDown">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="D3DChips_Dlg.qrc">
            <normaloff>:/D3DChips_Dlg/Resources/Stock-Index-Down-icon16.png</normaloff>:/D3DChips_Dlg/Resources/Stock-Index-Down-icon16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Bounding Box</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_bbxa">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_bbxb">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="QPushButton" name="pushButton_calculateBB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Calculate From Subsets</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="D3DChips_Dlg.qrc">
          <normaloff>:/D3DChips_Dlg/Resources/calculator-icon.png</normaloff>:/D3DChips_Dlg/Resources/calculator-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_bbya">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_bbyb">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>z:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_bbza">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_bbzb">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Culling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_culling">
        <property name="text">
         <string>Do Whole Object View Frustum Culling</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_subsetCulling">
        <property name="text">
         <string>Do Subset View Frustum Culling</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="D3DChips_Dlg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_updateFromDrawable</sender>
   <signal>clicked()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>updateFromDrawable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_calculateBB</sender>
   <signal>clicked()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>calculateBB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ssUp</sender>
   <signal>clicked()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>moveSsUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ssDown</sender>
   <signal>clicked()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>moveSsDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_bbxa</sender>
   <signal>editingFinished()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>bbEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_bbxb</sender>
   <signal>editingFinished()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>bbEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_bbya</sender>
   <signal>editingFinished()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>bbEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_bbyb</sender>
   <signal>editingFinished()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>bbEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_bbza</sender>
   <signal>editingFinished()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>bbEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_bbzb</sender>
   <signal>editingFinished()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>bbEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_subsets</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>ssSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_subsets</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>ssItemChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_culling</sender>
   <signal>clicked()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>cullingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_subsetCulling</sender>
   <signal>clicked()</signal>
   <receiver>Renderable_Dlg</receiver>
   <slot>cullingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mappingCellChanged(int,int)</slot>
  <slot>mappingCellEntered(int,int,int,int)</slot>
  <slot>updateFromDrawable()</slot>
  <slot>calculateBB()</slot>
  <slot>moveSsUp()</slot>
  <slot>moveSsDown()</slot>
  <slot>bbEdited()</slot>
  <slot>ssSelectionChanged()</slot>
  <slot>ssItemChanged(QListWidgetItem*)</slot>
  <slot>cullingChanged()</slot>
 </slots>
</ui>
